### DISCUSSION TITLE
ScriptReference/Mathf.Round

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#2] [Notes]
# ScriptReference/Mathf.Round

https://docs.unity3d.com/ScriptReference/Mathf.Round.html

**Comment by vertxxyz** [comment]
If you want to always round up from `0.5` increments, use the [`System.Math`](https://docs.microsoft.com/en-us/dotnet/api/system.math.round) class:
```csharp
(float) Math.Round(value, MidpointRounding.AwayFromZero)
```
